lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -122,7 +122,7 @@ sub test_network_interface {
         # Wait until guest's primary interface is back up
         script_retry("ping -c3 $guest", delay => 6, retry => 30);
         # Activate guest's secondary (tested) interface
-        script_retry("ssh root\@$guest ifup $nic", delay => 5, retry => 4);
+        script_retry("ssh root\@$guest ifup $nic", delay => 10, retry => 30, timeout => 120);
```

Comment:
Do we really need retry 30 times to try to bring up the interface? if you run tests in beijing subnet, dhcp ip run out sometime. In this case waiting for longer time may help, but this is not a normal situation and our test should not consider it.